tests/processes/BearRunningTest.py

```diff
@@ -253,13 +253,16 @@ def setUp(self):
         self.file_name_queue.put("invalid file")
         self.local_bear_list.append(LocalTestBear(self.settings,
                                                   self.message_queue))
-        self.local_bear_list.append("not a valid bear")
+        self.local_bear_list.append(SimpleGlobalBear({},
+                                                     self.settings,
+                                                     self.message_queue))
```

Comment:
why those changes? This tests that the routine handles wrong things like this so if it fails you should rather fix the method not the test right? This is also why coverage is going down.